onebusaway-android/src/amazon/java/org/onebusaway/android/map/googlemapsv2/BaseMapFragment.java

```diff
@@ -738,6 +739,8 @@ public void onLocationChanged(Location l) {
         if (firstFix) {
             // If its our first location (across all BaseMapFragments), move the camera to it
             setMyLocation(true, false);
+            //Notify the onFocusChangeListener when we get the first location
+            onFocusChanged(null,null,l);
```

Comment:
Per our discussion, let's remove this and pass the location we want the map to center on via an Intent to your Activity (when the user taps on "Send feedback" in the HomeActivity), which will then get passed to the BaseMapFragment.